src/stores/ViewportStore.ts

```diff
@@ -35,21 +34,24 @@ export default class ViewportStore extends Store<ViewportStoreState> {
     private readonly routeStore: RouteStore
     private readonly isSmallScreenQuery: () => boolean
 
-    constructor(routeStore: RouteStore, isSmallScreenQuery: () => boolean) {
+    constructor(routeStore: RouteStore, initialBBox: Bbox | null, isSmallScreenQuery: () => boolean) {
         super()
         this.routeStore = routeStore
         this.isSmallScreenQuery = isSmallScreenQuery
+        if (initialBBox) {
+            const updatedState = calculateLatLngFromBbox(this.state, initialBBox, isSmallScreenQuery())
+            super.setState(updatedState)
+        }
```

Comment:
@Janekdererste I wasn't sure how to do this, because `getInitialState` is called in the super-constructor and subclasses like `ViewportStore` have no chance to make use of the constructor parameters before `getInitialState` is called. Here I'd like to pass the `initialBBox` via the constructor and actually make use of it to set the initial state.

Reply:
Since the only way to alter a store's state should be through actions, I guess you need an action for this. Like UpdateBboxAction.

Initialize all the stores in ```index.ts``` then parse the bbox and issue an UpdateBboxAction to the Dispatcher.

Reply:
Ok, I did this in my last commit.